README.md

```diff
@@ -54,6 +63,8 @@ Set the Lambda Function environment variables as follows:
  SIGNALFX_API_PORT=[443]
  SIGNALFX_API_SCHEME=[https]
  SIGNALFX_SEND_TIMEOUT=milliseconds for signalfx client timeout [2000]
+ CONNECTION_LEASE_TIMEOUT=milliseconds for connection lease timeout [5000]
```

Comment:
can you make a note this and the config directly below it is for AspNetCore only